test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -1426,7 +1426,7 @@ await CliTester.Run(new RunConfiguration[]
              }, _output);
         }
 
-        [Theory]
+        [Theory(Skip = "Test is flakey")]
```

Comment:
Let's capture this test name somewhere (may be in a GH issue) so that we can come back and address it with other flaky tests.